src/lib.rs

```diff
@@ -496,3 +496,29 @@ impl<'ui> Ui<'ui> {
     pub fn menu<'p>(&self, label: ImStr<'p>) -> Menu<'ui, 'p> { Menu::new(label) }
     pub fn menu_item<'p>(&self, label: ImStr<'p>) -> MenuItem<'ui, 'p> { MenuItem::new(label) }
 }
+
+//Widgets: Combos 
+impl<'ui> Ui<'ui> {
+    pub fn combo<'p>(&self, label : ImStr<'p>, current_item : &mut i32, items : &mut Vec<ImStr<'p>>, items_count : i32, height_in_items : i32) -> bool{
+        let items_inner : Vec<*const c_char> = items.into_iter().map(|item| item.as_ptr()).collect();
+        unsafe {
+            imgui_sys::igCombo(label.as_ptr(), current_item, items_inner.as_ptr() as *mut *const c_char, items_count, height_in_items)
+        }
+    }
+
+    pub fn combo2<'p>(&self, label : ImStr<'p>, current_item :&mut i32, items : ImStr<'p>, height_in_items : i32) -> bool {
```

Comment:
Thanks for the explanation. For the `code2` function I thought that all underlying functions must be supported. I even tried to implement he one with callback namely 'combo3' using closures but I could not get it working.